File: Python/EstruturaDeRepeticao/exercicio_07.py
"""
Faça um programa que leia 5 números e informe o maior número.
"""

for i in range(1, 6):
    numero = int(input(f"Digite o {i}º número: "))
    if i == 1:
        maior = numero
    else:
        if numero > maior:
            maior = numero

print(f'O maior número foi {maior}.')

File: Java/EstruturaDeRepeticao/Exercicio07.java
// Faça um programa que leia 5 números e informe o maior número.
package EstruturaDeRepeticao;
import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite 5 números: ");

        int maior = Integer.MIN_VALUE;

        for (int i = 0; i < 5; i++) {
            int numero = scanner.nextInt();

            if (numero > maior) {
                maior = numero;
            }
        }

        System.out.println("O maior número é: " + maior);

        scanner.close();
    }
}
